### PR TITLE
Fix setup.py and requirements.txt

### PR DESCRIPTION
**Pull request by mapeveri**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,8 +27,8 @@
     description='The core for the Masonite ',
```
Comment (josephmancuso) [inline setup.py]: Also can you get rid of this trailing whitespace at the end?

File: setup.py [hunk 2]
```diff
@@ -26,8 +27,8 @@
...
     author='Joseph Mancuso',
     author_email='idmann509@gmail.com',
-    url='https://github.com/josephmancuso/masonite',  # use the URL to the github repo
+    url='https://github.com/MasoniteFramework/masonite',
     download_url='',
```
Comment (josephmancuso) [inline setup.py]: Can you get rid of the download_url? seems like we're not using it.

**Comment by mapeveri** [comment]
Check please!

**Comment by josephmancuso** [comment]
Thanks!

**Pull request merged**